app/src/main/java/de/westnordost/streetcomplete/MainActivity.java

```diff
@@ -344,11 +346,36 @@ private int getQuestTitleResId(Quest quest, Map<String,String> tags)
 				if(isConnected()) uploadChanges();
 				else              Toast.makeText(this, R.string.offline, Toast.LENGTH_SHORT).show();
 				return true;
+			case R.id.action_note:
+				createNote();
+				return true;
 		}
 
 		return super.onOptionsItemSelected(item);
 	}
 
+	@UiThread private void createNote()
+	{
+		if (mapFragment.getZoom() < ApplicationConstants.NOTE_MIN_ZOOM)
+		{
+			Toast.makeText(this, R.string.create_new_note_unprecise, Toast.LENGTH_LONG).show();
+		}
+		else
+		{
+			CreateNoteFragment f = new CreateNoteFragment();
+			Bundle args = new Bundle();
+			LatLon pos = mapFragment.getPosition();
+			args.putDouble(CreateNoteDialog.ARG_LAT, pos.getLatitude());
+			args.putDouble(CreateNoteDialog.ARG_LON, pos.getLongitude());
+			f.setArguments(args);
+
+			FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
+			ft.add(R.id.map_create_note_container, f, CREATE_NOTE);
```

Comment:
Couldn't the normal bottom sheet container be used? (Not sure right now, just looking at the diff)